README.md

```diff
@@ -63,11 +63,13 @@ Testing the Payroll Income or Document Income flows requires modifying the steps
 
 ### Transfer notes
 
-Using the Transfer endpoints requires you to be enrolled in the Transfer beta. Contact your Plaid Account manager or Plaid Sales to join the beta.
+To call Transfer endpoints, you will need either a payment profile token, or both an access token and an account id. 
 
-To call Transfer endpoints, you will need either a payment profile token, or both an access token and an account id. To obtain a payment profile token, call Transfer -> Create Payment Profile. To obtain an access token and account id, follow the [Making API calls in Sandbox](#making-api-calls-in-sandbox) steps above, then call Items -> Item Management -> Retrieve an Item's accounts. 
+The easiest method is to use an access token an account id. To obtain an access token and account id, follow the [Making API calls in Sandbox](#making-api-calls-in-sandbox) steps above, then call Items -> Item Management -> Retrieve an Item's accounts. 
 
-Once you have a payment profile token or an access token / account id pair, start by calling Transfer -> Authorize a transfer, then call Transfer -> Initiate a transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using Payment Profiles, delete the `account_id` and `access_token` fields before making the request.
+Once you have the access token / account id pair, start by calling Transfer -> Authorize a transfer, then call Transfer -> Initiate a transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using the account id and access token approach, delete the `payment_profile` field before making the request.
```

Comment:
```suggestion
Once you have the access token / account id pair, start by calling Transfer -> Authorize a Transfer, then call Transfer -> Initiate a Transfer. Note that you will need to update the request bodies for these endpoints to use only the account identifying mechanism you are choosing -- for example, if you are using the account id and access token approach, delete the `payment_profile` field before making the request.
```
nitL We seem to be leaning towards title case when specifying the endpoints to call.

Reply:
the actual text in Postman right now uses sentence case, so I'm going to leave it to match that